playwright_tests/E2E/staff-search.spec.ts

```diff
@@ -1,36 +1,33 @@
 import { test, expect } from '@playwright/test';
-import { signIn, signOut } from "./steps/login-steps";
-import { clickToStaffPage, fillSearchBox } from "./steps/staff-steps";
+import { signIn, signOut } from "../steps/login-steps";
+import { clickToStaffPage } from "../steps/staff-steps";
 
 test('Simplified search results', async ({ page }) => {
     await signIn(page, "STAFF_ADMIN");
     await clickToStaffPage(page);
-  
     console.log("Using simple search");
-    await fillSearchBox(page);
+    await page.locator('#main-content').getByRole('textbox').click();
+    await page.locator('#main-content').getByRole('textbox').fill('xui');
```

Comment:
If that is the login, please parameterise. From time to time we have to change the logins because some service cleans something up.